src/Components/ListOfArticles.jsx

```diff
@@ -1,37 +1,60 @@
 import React, { Component } from 'react';
 import axios from 'axios';
 import { Grid, Container } from 'semantic-ui-react'
+import { Redirect } from 'react-router-dom'
 
 class ListOfArticles extends Component {
-  state = {
-      articles: []
+  constructor(props) {
+    super(props);
+    this.state = {
+      articles: [],
+      id: '',
+      redirect: false
     }
+  }
 
   componentDidMount() {
-    axios.get('http://localhost:3000/api/v1/articles').then(response => {
+    axios.get('/api/v1/articles').then(response => {
       this.setState({ articles: response.data.entries });
     });
   }
 
+  handleClick = (e) => {
+    this.setState({
+      redirect: true,
+      id: e.currentTarget.id
+    })
+  }
+
   render() {
-    let articleList = this.state.articles.length ? (
-      <div>
-        {this.state.articles.map(article => {
-          return (
-            <div id={`id_${article.id}`} key={article.id}>
-              <img alt="article logo" id={`photo_${article.id}`} src={article.image} width="200" height="100" />
-              <h1 id={`title_${article.id}`}>{article.title}</h1>
-              <h3 id={`ingress_${article.id}`}>{article.ingress}</h3>
-              <br />
-            </div>
-          )
-        })}
-      </div>
-    ) : (
-      <h2>There are no articles at the moment. You can be the first to post your own article and become a neighborhood journalist!</h2>
-    )
+    let articleList
+    if (this.state.redirect === true) {
+      return <Redirect to={{
```

Comment:
Why do you introduce the `Redirect` component here instead of using `Link` as in other parts of your code.

IMO, that would be cleaner and you could remove the if statement that you added to this component

Reply:
Changed to Link